pkg/common/vars.go

```diff
@@ -58,3 +90,13 @@ func SkipFile(filename string) bool {
 	}
 	return false
 }
+
+// IsBinary returns true if the file extension is in the binaryExtensions list.
+func IsBinary(filename string) bool {
+	for _, ext := range binaryExtensions {
+		if strings.TrimPrefix(filepath.Ext(filename), ".") == ext {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
**Nit:** In another PR we should do a map lookup (or even better?, a switch case).

Reply:
nah, i'm fix this right now lol. I like the map too. I'll also post benchies.

Reply:
![Screenshot 2023-12-15 at 10 41 03 AM](https://github.com/trufflesecurity/trufflehog/assets/21311841/ff99dcad-9490-4041-b66f-b36b7122f39f)

#microptimizations 🥳